pyanaconda/errors.py

```diff
@@ -117,7 +117,7 @@ def _storage_install_handler(self, exn):
 
     def _storage_reset_handler(self, exn):
         message = (_("There is a problem with your existing storage "
-                     "configuration. "
+                     "configuration or your initial settings. "
```

Comment:
"initial settings" seems a bit ambiguous to me to what initial settings refers to - intial settings of the *installation* or something else ?

So I think something like "initial installation settings" or even "initial settings (kickstart, etc.)" could be more clear.

Reply:
Hm, true... How is this message displayed? It's the "some errors with your setup" yellow bar with "click here for details", rigth? If so, there's enough space so I would go for explicit list of places things might be specified.

Reply:
Message from blivet is also part of the error message so we can be more specific there (feel free to suggest a better message in the blivet PR https://github.com/storaged-project/blivet/pull/896).